TestApp/.env.template

```diff
@@ -1 +1,3 @@
-CONNECTION_STRING="event_hub_connection_string"
\ No newline at end of file
+CONNECTION_STRING="event_hub_connection_string"
```

Comment:
Since we have specific connection string names for the other two files can we change the one on `line 1` to also be specific?

Reply:
no